lib/layout/box_layout.dart

```diff
@@ -200,6 +212,16 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
 
   }
 
+  Widget _buildListView(List<Widget> listViewChildren) {
+    return flutter.ListView.builder(
+        scrollDirection: Axis.vertical,
+        itemCount: listViewChildren.length,
+        shrinkWrap: true,
+        itemBuilder: (BuildContext context, int index) {
```

Comment:
Can you run a test of say 100 widgets. All 100 widgets will be constructed and called their initState(), but only the visible one in the ListView will call their buildWidget() method?

Reply:
I think it should work like that but if it doesn't we have to do proper paging.